examples/in-composition/composition.yaml

```diff
@@ -182,10 +192,12 @@ spec:
           forProvider:
             chart:
               name: wordpress
-              repository: https://turkenh.github.io/bitnami-charts
+              repository: https://charts.bitnami.com/bitnami
```

Comment:
No, the Bitnami chart was getting to the point that you describe and failing there. I changed this because I thought at first glance that your fork no longer existed. I think I was mistaken; I'll revert this line.